core/krylovTimeEvolver.cpp

```diff
@@ -494,46 +427,58 @@ krylovReturn* krylovTimeEvolver::timeEvolve()
             cblas_zgemv(CblasColMajor, CblasNoTrans, m, m, &one, schurvector, m, tmpKrylovVec1, 1, &zero, tmpKrylovVec2, 1);
             cblas_zgemv(CblasColMajor, CblasNoTrans, Hsize, m, &one, V->values, Hsize, tmpKrylovVec2, 1, &zero, sampledState, 1);
 
-            sample();
+			try {
+				sample();
+			} catch (requestStopException& e)
+			{
+				logger.log_message(krylovLogger::ERROR, e.what());
+				delete[] eigenvalues; delete[] schurvector;
+				delete V; delete H;
+				if (progressBar){
+					stop_printing = true;
+					pBThread.join();
+				}
+				statusCode = 1000;
```

Comment:
Yes, definitely. I didn't think too much about it. As you pointed out, the time evolution is perfectly alright up to this point (and would be most likely also after that).  If observables have a different measure of success it's not of TimeEvolver business.
Besides of the loss of previous errors/warnings in that case. This will be addressed in some future release when we rework the statusCode and error handling. 

I assigned the creative number 3 then to this case for now. 
41434eb5